gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/schema/source/DefaultAuthSourceService.java

```diff
@@ -106,6 +109,7 @@ public Optional<AuthSource> getAuthSourceFromRequest(HttpServletRequest request)
             service = getService(AuthSourceType.CLIENT_CERT);
             authSource = service.getAuthSourceFromRequest(request);
         }
+        authSource.ifPresent(source -> log.debug(logMessage, request.getRequestURI(), source.getType().toString()));
```

Comment:
```suggestion
        authSource.ifPresent(source -> log.debug(logMessage, request.getRequestURI(), source.getType()));
```

Reply:
Oh yes, the conversation to string would be implicitly done by the log.debug. I will remove it